lib/cc/engine/analyzers/violation.rb

```diff
@@ -48,7 +48,7 @@ def other_sexps
           @other_locations ||= sorted_hashes.drop(1)
         end
 
-        def name
+        def check_name
```

Comment:
Should identical vs. similar factor into the fingerprint? Maybe out of scope for this?